### PR TITLE
Email multipart

### PR DESCRIPTION
**Pull request by soyapark-kaist**
Resolve a bug which occurs at a following situation:
when a user replies to the thread with an attachment, other users who is using apple mail can't see the previous conversation at the thread.

### CODE REVIEW

File: browser/templates/murmur/add_members.html [hunk 1]
```diff
@@ -5,7 +5,7 @@ <h3>Add New Members to <span id="group-name">{{ group_info.name }}</span></h3>
 	<hr />
 	
 	<form id="new-members-form">
-		<span class="strong">Emails Separated by Commas : </span> <br />
+		<span class="strong">Emails Separated by Seperators (e.g. , ; | /) : </span> <br />
```
Comment (amyxzhang) [inline browser/templates/murmur/add_members.html]: "Seperators"  -> "Separators"

File: engine/main.py [hunk 2]
```diff
@@ -528,7 +528,11 @@ def add_members(group_name, emails, add_as_mods, user):
             membergroup = MemberGroup.objects.get(group=group, member=user)
             is_admin = membergroup.admin
         if is_public or is_admin:
-            email_list = emails.strip().lower().split(',')
+            emails_txt = emails.strip().lower()
+            email_serators = [emails_txt.split(','), emails_txt.split(';'), emails_txt.split('|'), emails_txt.split('/')]
+            email_serators_len = [len(x) for x in email_serators]
+            seperator_index = email_serators_len.index(max(email_serators_len))
+            email_list = email_serators[seperator_index]
```
Comment (amyxzhang) [inline engine/main.py]: this doesn't work if users mix the possible separators in the same string.

File: schema/models.py [hunk 3]
```diff
@@ -284,7 +284,7 @@ class UserProfile(AbstractBaseUser):
 	is_admin = models.BooleanField(default=False)
 	date_joined = models.DateTimeField(auto_now=True)
 	hash = models.CharField(max_length=40, default="")
-	imapAccount = models.ForeignKey('ImapAccount', blank=True, null=True)
+	#imapAccount = models.ForeignKey('ImapAccount', blank=True, null=True)
```
Comment (amyxzhang) [inline schema/models.py]: can this be deleted? any reason why we're keeping the comment in?

File: browser/templates/murmur/add_members.html (hunk 1 above)
Comment (soyapark-kaist) [reply browser/templates/murmur/add_members.html]: Done!

File: schema/models.py (hunk 3 above)
Comment (soyapark-kaist) [reply schema/models.py]: Done!

File: engine/main.py (hunk 2 above)
Comment (soyapark-kaist) [reply engine/main.py]: Yes. That's intentional. People don't mix up the separators within one list of email addresses

Comment (soyapark-kaist) [reply engine/main.py]: Done!

**Comment by soyapark-kaist** [comment]
Done!

**Pull request merged**